src/__snapshots__/Button.test.js.snap

```diff
@@ -0,0 +1,9 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Button renders a blank button 1`] = `"<styled.button onClick={[undefined]} />"`;
+
+exports[`Button renders a button with text 1`] = `
+"<styled.button onClick={[undefined]}>
```

Comment:
I'd probably pass an onClick to each of the tests, as it would be a required prop

Reply:
Would that mean we'd have to mock a function for each one? or just use some placeholder?

Reply:
No, so you normally only use a jest.fn() mock when you are testing to see if that function was called. But when you're just providing a function and dont care if it's called or not you can just give it the value `() => {}`